drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/models/ScenarioGridModel.java

```diff
@@ -515,6 +518,58 @@ public void clearSelections() {
         selectedColumn = null;
     }
 
+    /**
+     * It forces a <code>internalRefreshWidth</code> refresh
+     */
+    public boolean forceRefreshWidth() {
```

Comment:
@yesamer 
Please check it is really needed - columns width are already automatically managed, and persisting them on our model should happen only at "save" invocation

Reply:
@gitgabrio In my opinion this is still required, because it's necessary to launch AutoResize process after loading of column widths from scesim file.